src/runtime/server/utils/proxy-csrf.ts

```
@@ -0,0 +1,41 @@
+import { createError, getCookie, getHeader, setCookie } from 'h3'
+import type { H3Event } from 'h3'
```

ESLint

[error] Expected "h3" (type-import) to come before "h3" (value-external).

(perfectionist/sort-imports)